hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1469,6 +1542,18 @@
               "chip_sw_kmac_mode_kmac_jitter_en",
               "chip_sw_sram_ctrl_main_scrambled_access_jitter_en"]
     }
+    {
+      name: jitter_reduced_freq
+      tests: ["chip_sw_clkmgr_jitter_reduced_freq",
+              "chip_sw_flash_ctrl_ops_jitter_en_reduced_freq",
+              "chip_sw_flash_ctrl_access_jitter_en_reduced_freq",
+              "chip_sw_otbn_ecdsa_op_irq_jitter_en_reduced_freq",
+              "chip_sw_aes_enc_jitter_en_reduced_freq",
+              "chip_sw_hmac_enc_jitter_en_reduced_freq",
+              "chip_sw_keymgr_key_derivation_jitter_en_reduced_freq",
+              "chip_sw_kmac_mode_kmac_jitter_en_reduced_freq",
+              "chip_sw_sram_ctrl_main_scrambled_access_jitter_en_reduced_freq"]
```

Comment:
Is this missing `chip_sw_flash_init_reduced_freq`?

Reply:
Added to the list, see my note above.